dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -245,4 +245,33 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             LOG_IF_FAILED(::CoRevokeClassObject(static_cast<DWORD>(token.Cookie())));
         }
     }
+
+    bool PushNotificationManager::IsBackgroundTaskBuilderAvailable()
+    {
+        auto builder{ BackgroundTaskBuilder() };
+        auto builder5{ builder.try_as<winrt::IBackgroundTaskBuilder5>() };
+        return builder5 != nullptr;
+    }
+
+    bool PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions const& options)
+    {
+        THROW_HR_IF(E_INVALIDARG, options == PushNotificationRegistrationOptions::Undefined); // WI_IsFlagSet does not allow 0 as flag.
```

Comment:
Remove comment (it's redundant - can't test bits when you provide no bits to test...)

In theory you could `WI_AreAllFlagsClear(options)`? Or `WI_AreAllFlagsClear(options, static_cast<int>(~0))`?
But the `options==Undefined` is clearer